pkg/api/server/docs.go

```diff
@@ -4,6 +4,24 @@
 // only as experimental as this point.  The endpoints, parameters, inputs, and
 // return values can all change.
 //
+// To start the service and keep it running for 5,000 seconds:
+//    podman system service -t 5000 &
```

Comment:
Should explicitly mention 0 timeout = never stops

Reply:
Maybe also add an example for 'HTTP', e.g.
```
$ podman system service --timeout 0 tcp:127.0.0.1:8080
```